runner.py

```diff
@@ -106,18 +110,17 @@ def generate_cases(params):
             for dataset in params_set['dataset']:
                 if dataset['source'] in ['csv', 'npy']:
                     train_data = dataset["training"]
-                    test_data = dataset["testing"]
-
                     file_train_data_x = train_data["x"]
-                    file_train_data_y = train_data["y"]
-                    file_test_data_x = test_data["x"]
-                    file_test_data_y = test_data["y"]
                     paths = f'--file-X-train {file_train_data_x}'
                     if 'y' in dataset['training'].keys():
+                        file_train_data_y = train_data["y"]
```

Comment:
`file_train_data_y` is used on line 130 and now might be unassigned.
I guess, need to change line 130 to `if not utils.is_exists_files([file_train_data_x]):`

Reply:
Make sense, thanks! Applied